Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1804,6 +1804,29 @@ class BrowserViewController: UIViewController {
             }
             activities.append(requestDesktopSiteActivity)
             
+            if let metadata = tab?.pageMetadata, !metadata.feeds.isEmpty {
```

Comment:
I assume this is safe and doesn't persist any data until user interaction? 
Thinking of browsing in private mode and persisting metadata by accident

Reply:
Right, the metadata is only preserved as long as the share sheet and tab is visible as its passed into the activity thats handed to the `UIActivityViewController`